engine/say.py

```diff
@@ -64,7 +64,7 @@ def insayne(text, add_newline=True, insanity=None):
     if add_newline:
         adventurelib.say("")
     if insanity is None:
-        insanity = G.player.insanity.value
+        insanity = G.player.insanity.current_value
```

Comment:
The distinction between `value` and `current_value` isn't clear to me. Could you choose different variable names that elucidate the difference between these values?